tests/e2e/features/edit_record.feature

```diff
@@ -26,6 +25,8 @@ Feature: Edit record
         When I fill the "title_field" field with "Changed title EN"
         And I click the "save_button" element
 
+        And I wait for "1" seconds
```

Comment:
why wait?

Reply:
I wanted to ask you about that. For some reason this test breaks, while it used to work. Travis gives us this: 

```
2) Scenario: As an Admin I want to change title of a record in another language # features/edit_record.feature:19
   ✔ Before # node_modules/kakunin/dist/web/cucumber/hooks.js:95
   ✔ Given I am logged in as "admin" # step_definitions/login.js:4
   ✔ When I visit the "edit_record" page with parameters: # node_modules/kakunin/dist/step_definitions/navigation.js:14
       | id | 1 |
   ✔ Then I wait for "title_field" element to appear # step_definitions/steps.js:6
   ✔ When I fill the "title_field" field with "Changed title EN" # step_definitions/steps.js:19
   ✔ And I click the "save_button" element # node_modules/kakunin/dist/step_definitions/elements.js:73
   ✔ And I wait for "1" seconds # node_modules/kakunin/dist/step_definitions/debug.js:6
   ✖ And I click the "lang_select" element # node_modules/kakunin/dist/step_definitions/elements.js:73
       Error: Error, after scrolling the element "lang_select" is still not clickable.
           at /home/travis/build/bolt/four/tests/e2e/node_modules/cucumber/lib/user_code_runner.js:95:19
           at Generator.throw (<anonymous>)
           at asyncGeneratorStep (/home/travis/build/bolt/four/tests/e2e/node_modules/cucumber/lib/user_code_runner.js:18:103)
           at _throw (/home/travis/build/bolt/four/tests/e2e/node_modules/cucumber/lib/user_code_runner.js:20:291)
           at <anonymous>
```

Reply:
I've updated that, but I can't get this same test to work for both my open PR's. It breaks on changes introduced in #295 and i don't understand why they passed in that PR, but break here.. Could you help me out, here?